pages/common/zapier.md

```diff
@@ -0,0 +1,33 @@
+# zapier
+
+> Create, automate, and manage zapier integrations.
+> Some subcommands such as `build`, `init`, `scaffold`, `push`, `test`, etc. have their own usage documentation.
+> More information: <https://github.com/zapier/zapier-platform/blob/main/packages/cli/docs/cli.md>.
+
+- Connect to your Zapier account:
```

Comment:
```suggestion
- Connect to an Zapier account:
```

Reply:
Should be `a Zapier account` instead of an but good catch